### ISSUE TITLE
WCAG 2.2: Scroll bar within combobox is difficult to activate, due to width (15px wide)

### ISSUE THREAD

**Issue by dav-idc** [alphagov/accessible-autocomplete#604]
## Description of the issue

When the accessible autocomplete combobox (the pop-over area with choices to select) is activated, its associated scroll bar is fairly thin.

## Steps to reproduce the issue

You can test it out on the accessible autocomplete examples demo: https://alphagov.github.io/accessible-autocomplete/examples/

Here's a screenshot of one of the autocomplete inputs within that demo. I've typed 'a' in the input, and then the combobox appears. It shows a series of 'a' named countries to choose from. The list overflows the combobox area, so a scroll bar appears on the right side of the combobox area.

![screenshot of an accessible autocomplete demo with the letter 'a' in the input and a scrollable list of 'a' countries in the combobox.](https://github.com/alphagov/accessible-autocomplete/assets/98318778/c456807f-43d9-42a9-8133-163140b6092d)

## Actual vs expected behaviour

Actual behaviour: the scroll bar has a hit-target width of 15px on Chrome 118, on MacOS Ventura.

Expected behaviour: [WCAG 2.2 AA target size (minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum) recommends hit targets to be minimum 24px by 24px.

There's an exception for elements under user agent control (things set by the browser or operating system), which would apply in this case.

So this isn't a WCAG failure, but nonetheless, it is difficult to select the scrollbar.

## Recommended action
I recommend we:

- [x] Investigate whether we're currently modifying the accessible autocomplete scroll bar with CSS (I don't believe we are)
- [x] Decide whether we would want to modify it to set a minimum width of 24px
- [ ] Test how this interacts with various browsers

## Environment (where applicable)

I only tested the scrollbar width on one setup.

- Operating system: MacOS Ventura
- Browser: Chrome 118
- GOV.UK Frontend Version: V4.7

**Comment by edwardhorsford** [comment]
Presumably the same applies for *any* scrollbar that might appear in a government service - anything that's got overflow / internal scrolling? If so, I'd suggest any implementation might want to consider tackling scroll bars in general.

Note: as an aside, having very long scrolling lists within the autocomplete isn't ideal - that's why the default requires two characters to be typed before results are returned - to try to reduce the initial number of results returned.

**Comment by owenatgov** [comment]
Following a discussion with the team, we are going to **close this with the intent to not do it**. Our rationale for this is:

1. Because the scrollbar is a user agent style, there's a chance that users will customise it. If we apply styling over top of this, there's a risk that we'll undermine that option for customisation and take power away from users. The issue here lies with browser vendors themselves and because we're technically passing this criterion we have the affordance for the problem to be solved at that level.
2. As Ed pointed out above, this doesn't just apply to the accessible autocomplete but scrollbars across services using govuk-frontend. Several of our components could trigger an overflow and display a scrollbar within them and arguably pages on govuk services are longer than most viewports and themselves have scrollbars. Tackling one and not all could create dissonance and the scope here is tremendous. That's not mentioning the headache of how we would apply this across our codebase. We think that this would be a lot of effort for very little return and a sizeable amount of risk to users as per point 1.
